src/components/Navigator/Navigator.tsx

```diff
@@ -1,19 +1,122 @@
 import React, { useEffect, useState } from "react";
 import { styled } from "@stitches/react";
+import NavigatorItem from "components/NavigatorItem/NavigatorItem";
+import sampleParsedVtt from "../../manifests/sampleParsedVtt";
 
-interface Props {
+interface NavigatorProps {
   currentTime: number;
   tracks: object;
 }
+interface NavigatorTabProps {
+  label: string;
+  active: boolean;
+}
+interface NavigatorBodyInnerProps {
+  currentTime: number;
+  items: Array<object>;
+}
 
-export const Navigator: React.FC<Props> = ({ currentTime }) => {
-  return <NavigatorWrapper>{currentTime}</NavigatorWrapper>;
+export const Navigator: React.FC<NavigatorProps> = ({ currentTime }) => {
+  return (
+    <NavigatorWrapper>
+      <NavigatorHeader>
+        <NavigatorTab label="Camptium" active={true} />
+        <NavigatorTab label="Mauris" active={false} />
+        <NavigatorTab label="Tristique" active={false} />
+      </NavigatorHeader>
+      <NavigatorBody>
+        <NavigatorOutput>
+          <NavigatorBodyInner
+            items={sampleParsedVtt.output}
+            currentTime={currentTime}
+          />
+        </NavigatorOutput>
+      </NavigatorBody>
+    </NavigatorWrapper>
+  );
+};
+
+const NavigatorTab: React.FC<NavigatorTabProps> = ({ label }) => {
+  return <NavigatorTabWrapper>{label}</NavigatorTabWrapper>;
+};
+
+const NavigatorBodyInner: React.FC<NavigatorBodyInnerProps> = ({
+  items,
+  currentTime,
+}) => {
+  return items.map((cue) => {
+    return (
+      <NavigatorItem
```

Comment:
Also getting some TypeScript complaints here...  interesting to know that it will still compile with these complaints...I'll look into that.

Reply:
Yeah, this definitely need some work. I think it compiles because it's a JS map and not a TS map. I was banging my head against apply TS here and just went with what I knew to get it to map without a fail.